code/modules/reagents/chemistry/reagents/drinks/drink_reagents.dm

```diff
@@ -1142,3 +1142,20 @@
 			drinker.adjust_hallucinations(60 SECONDS * REM * seconds_per_tick)
 
 	return ..()
+
+/datum/reagent/consumable/t_letter
+	name = "T"
+	description = "You expected to find this in a soup, but this is fine too."
+	color = "#583d09" // rgb: 88, 61, 9
+	taste_description = "one of your 26 favorite letters"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/t_letter/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	if(!ishuman(affected_mob) || !HAS_MIND_TRAIT(affected_mob, TRAIT_MIMING))
```

Comment:
I know plenty of other, mime-related drinks and otherwise have these human-only checks, but if my memory doesn't fail me, that's because miming used to be a human var before a trait. This is just a suggestion, however:
```suggestion
	if(!HAS_MIND_TRAIT(affected_mob, TRAIT_MIMING))
```

Reply:
I mean I could try swapping it tomorrow but idk if removing that safety net would be the best idea

Reply:
Well, the procs are already defined at `/carbon` so it SHOULD be safe. If any error is thrown, it's a fault in the code of `/carbon`, not of this drink.

Reply:
Alright done, also I swapped the boost from an increased heal (`-4`) to an increased probability of proccing the heal, which should result in a similar amount of healing, but more consistent